submissions/EmmanuelIdiaghe/php-database/admin.php

```diff
@@ -0,0 +1,72 @@
+<?php
+#Variable Declaration
+$notSet = !(isset($_COOKIE["show"])); //cookie not set
+
+
+# Function that sets cookie
+function load() {
+    //SetUp Connection
+    global $pdo;
```

Comment:
I don't see the need of setting `$pdo` here as `global`

Reply:
Also, extract your db connection into a different file, so you can reuse it else where

Reply:
The $pdo variable has to be global here because the load() function scope is outside the scope of the connect.php file where it was declared. Making it non-global flags an error on the query() function.

And thanks, I've extracted the connection query to a separate file.

Reply:
Here's a screenshot of the error message with not making $pdo global

![Screenshot (80)](https://user-images.githubusercontent.com/29339127/107527060-c2208e00-6bb8-11eb-8380-3d3e615c7c36.png)

Reply:
I don't think there is a need for that. You already include connect.php here, which makes it easy to access `$pdo` from within the load function

Reply:
Here's the screenshot of the error I get without making $pdo global

![Screenshot (81)](https://user-images.githubusercontent.com/29339127/107621695-f2b70500-6c56-11eb-9877-65caaad1cc26.png)